### PR TITLE
Add maual testing guide

### PR DESCRIPTION
**Pull request by michaellzc**
### Checklist

- [x] Follow [TEST](https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/main/TEST.md) to perform testing

### Test plan

this is a test

close https://github.com/sourcegraph/sourcegraph/issues/32175

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #61** <a href="https://app.graphite.dev/github/pr/sourcegraph/deploy-sourcegraph-helm/61" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### Checklist
+
+- [ ] Follow [TEST](https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/main/TEST.md) to perform testing
```
Comment (caugustus-sourcegraph) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
- [ ] Follow the [manual testing process](https://github.com/sourcegraph/deploy-sourcegraph-helm/blob/main/TEST.md)
```

File: TEST.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Sourcegraph Helm Chart Test Guide
+
+This is a temporary test guide before we have more throughout automated integration tests.
```
Comment (caugustus-sourcegraph) [inline TEST.md:3]:
```suggestion
This is a temporary test guide before we have more thorough automated integration tests.
```

File: TEST.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  create: false
+  name: standard
+
+# Disable resources requests/limits
+sourcegraph:
+  localDevMode: true
+# More values to be added in order to test your change
+```
+
+It's a good idea to inspect the rendered manifest to catch things that look off.
```
Comment (caugustus-sourcegraph) [inline TEST.md:43]: More recommendations:
- Perform a diff of the rendered helm manifests before and after your change
- Test both enabled and disabled toggles

**Review by caugustus-sourcegraph — APPROVED** [review]

**Review by caugustus-sourcegraph — APPROVED** [review]

**Pull request merged**